### PR TITLE
Include requirements of tests in generated license notice

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
# Description

Tests are our delivery (as part of our devContainer). Therefore the dependencies of the tests shal be part of our license notice.

## Issue ticket number and link

AB#17318

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

**Review by MP91 — APPROVED** [review]
lgtm

**Review by BjoernAtBosch — COMMENTED** [review]
LGTM

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -80,29 +80,30 @@ repos:
...
-          - --comment-style
-          - //
-      - id: insert-license
-        files: "Dockerfile.*"
-        args:
-          - --license-filepath
-          - license_header.txt
-          - --comment-style
-          - "#"
+# Commented out due to insufficient handling of year range in coyright statements
```
Comment (kse3hi) [inline .pre-commit-config.yaml:83]: Please remove it, if it is not necessary. Copy can be recovered from git history.

File: requirements.in [hunk 2]
```diff
@@ -22,7 +22,7 @@
 ##########################################
 grpcio-tools==1.57.0
```
Comment (kse3hi) [inline requirements.in:23]: grpcio-tools shall be align with grpcio itself to avoid incompatibility issues.
https://pypi.org/project/grpcio-tools/

**Review by kse3hi — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (kse3hi) [reply .pre-commit-config.yaml:83]: Maybe create a task to fix it or improve in future

**Review by kse3hi — CHANGES_REQUESTED** [review]

File: requirements.in (hunk 2 above)
Comment (BjoernAtBosch) [reply requirements.in:23]: This is a bit problematic as grpcio-tools are defined in template and grpcio in SDK.
Why is this at all?
Could we move grpcio-tools to SDK, too? (Is it directly used in template?)

Comment (MP91) [reply requirements.in:23]: Do we need them at all in the template? As far as I see, the tools provide the compiler, but we only compile the protofiles in the sdk...

File: .pre-commit-config.yaml (hunk 1 above)
Comment (MP91) [reply .pre-commit-config.yaml:83]: we already have a bug-ticket for this issue

**Review by kse3hi — APPROVED** [review]
LGTM

**Pull request merged**